src/components/Identity/processIdSyncs.js

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { assign, cookie, fireDestinations } from "../../utils";
+
+const millisecondsPerHour = 60 * 60 * 1000;
+
+const getControlObject = () => {
+  return JSON.parse(cookie.get("adobeIdSyncControl") || "{}");
+};
+
+const setControlObject = obj => {
+  cookie.set("adobeIdSyncControl", JSON.stringify(obj), {
+    expires: 6 * 30 // 6 months
+  });
+};
+
+export default ({ destinations, config, logger }) => {
+  if (config.idSyncsEnabled === undefined || config.idSyncsEnabled) {
```

Comment:
Another place where you should be able to use destructuring defaults if you want:

```
const {
  idSyncsEnabled = true
} = config;
```